osgar/drivers/rosmsg.py

```diff
@@ -561,7 +561,7 @@ def slot_raw(self, timestamp, data):
         #if frame_id.endswith(b'camera_depth_frame'):
         #    import pdb
         #    pdb.set_trace()
-        if frame_id.endswith(b'/base_link/camera_front') or frame_id.endswith(b'camera_color_optical_frame'):
+        if frame_id.endswith(b'/base_link/camera_front') or frame_id.endswith(b'/base_link/camera_down') or frame_id.endswith(b'camera_color_optical_frame'):
```

Comment:
this is a bit going "out of control" - so maybe rospy next would be depth & RGB image data?? I.e. used named topics -> different JSONs for different robots

Reply:
:+1: It would also help with matching color image to depth data.